frontend/src/router.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Switch, Redirect } from 'react-router-dom';
+import MapView from './pages/MapView';
+import AccessRequestPage from './pages/AccessRequestPage';
+import EditUserPage from './pages/EditUserPage';
+import Login from './pages/Login';
+import AccessDenied from 'pages/AccessDenied';
+import SubmitProperty from 'pages/SubmitProperty';
+import ManageUsers from 'pages/ManageUsers';
+import ManageAccessRequests from 'pages/ManageAccessRequests';
+import { Claims } from 'constants/claims';
+import AppRoute from 'utils/AppRoute';
+import PublicLayout from 'layouts/PublicLayout';
+import AuthLayout from 'layouts/AuthLayout';
+
+const AppRouter: React.FC = () => {
+  return (
+    <Switch>
+      <Redirect exact from="/" to="/login" />
```

Comment:
Add an explicit `<Redirect>` from the root to `/login`. Before it was working because the login route was put first and got matched first. However that could lead to bugs if a developer puts another route on top. 

With the redirect we are explicit on what the default route should be.